zendesk_api.gemspec

```
@@ -13,6 +13,14 @@ Gem::Specification.new do |s|
   s.description = 'Ruby wrapper for the REST API at https://www.zendesk.com. Documentation at https://developer.zendesk.com.'
   s.license     = 'Apache License Version 2.0'
 
+  s.metadata = {
+    'bug_tracker_uri' => 'https://github.com/zendesk/zendesk_api_client_rb/issues',
+    'changelog_uri' => 'https://github.com/zendesk/zendesk_api_client_rb/blob/master/CHANGELOG.md',
```

Alternatively, rather than the `master` change log, we could link to the change log at the released tag.

```suggestion
    'changelog_uri' => 'https://github.com/zendesk/zendesk_api_client_rb/blob/v#{s.version}/CHANGELOG.md',
```

---

That is probably far more useful, thanks!